### ISSUE TITLE
Hard-coded Australian-centered values 

### ISSUE THREAD

**Issue by niconoe** [AtlasOfLivingAustralia/collectory-plugin#51]
Hi guys,

While exploring the ALA toolkit for european use, I was asked in some forms what was the australian state holding a given collection. 

It seems there are some hard-coded values in src/groovy/au/org/ala/collectory/ProviderGroup.groovy for example (`statesList`, `networkTypes`, ...). IMHO, now that the project is used at a global scale, these things should be made easily configurable (preferably via the admin interface, but an easy to override config file would also help).

Thanks,

**Comment by djtfmartin** [comment]
I agree. These lists should be moved either to external config files or into the database.
